internal/content/content.go

```diff
@@ -112,8 +139,16 @@ func copyHtmlHeaderFooter() error {
 }
 
 func convertNodeToNewFormat(parentPath string, metadataPtr, nodePtr *map[string]interface{}) error {
-	createNodeMetadata(parentPath, metadataPtr, nodePtr)
-	convertNodeContent(parentPath, metadataPtr, nodePtr)
+	err := convertNodeContent(parentPath, metadataPtr, nodePtr)
+	if err != nil {
+		log.Println(err.Error())
+		return nil
+	}
+	err = createNodeMetadata(parentPath, metadataPtr, nodePtr)
+	if err != nil {
+		log.Println(err.Error())
+		return nil
+	}
 	nodeFileName, err := getNodeFileName(*nodePtr)
 	if err != nil {
 		return err
```

Comment:
method's result is not checked anywhere
convertNodeToNewFormat
shoudl it stop execution on node file error ?